setup.py

```diff
@@ -36,9 +36,13 @@
 
 EXTRAS_REQUIRE = {
     'doc': [
-        'sphinx==1.8.5',
-        'sphinx_rtd_theme>=0.5.0',
+        'sphinx==3.2',
+        'jinja2<3.1',
+        'nbsphinx==0.8.8',
+        'nbsphinx-link==1.3.0',
```

Comment:
Are the two nbsphinx packages actually needed here?  Doesn't seem like the docs use any notebooks.

Reply:
Addressed in commit https://github.com/NREL/Panel-Segmentation/pull/15/commits/ddf30adf67b87182aedaf527ba4febc0b1cbab9f, I removed these guys (they were initially copied over from rdtools